lib/oper1.g

```diff
@@ -1090,3 +1090,13 @@ InstallMethod( ViewObj,
     [ IS_OBJECT ],
     0,
     PRINT_OBJ );
+
+BIND_GLOBAL( "CHECK_REPEATED_ATTRIBUTE_SET", function(obj, name, val)
+    if CurrentAssertionLevel >= 2 then
+        if not IsBound(obj!.(name)) then
+            ErrorNoReturn("Attribute ", name, " of ", obj, " is marked as assigned, but it has no value");
+        elif obj!.(name) <> val then
+            ErrorNoReturn(name, " of ", obj, " already set to ", obj!.(name), ", cannot be changed to ", val);
```

Comment:
I wouldn't print the old and new values: doing so can lead to errors itself; and it can also simply flood the screen, or cause all kinds of other problems. Inspecting these values if desired is what we have the break loop for. I'd instead give the relevant values suggestive names and indicate them in the error message, e.g. like so:
```suggestion
            ErrorNoReturn("Attribute ", name, " of <obj> already set to <oldval>, cannot be changed to <val>);
```
Then of course a local variable `oldval` should be added and set to `obj!.(name)`. 

(I'd also add the `"Attribute ", ` lead to the message, to match the other message)